plugins/pulp_rpm/plugins/importers/yum/utils.py

```diff
@@ -102,6 +111,67 @@ def strip_ns(element, uri=None):
         strip_ns(child, uri)
 
 
+def fake_xml_element(repodata_snippet):
+    """
+    Wrap a snippet of xml in a fake element so it can be coerced to an ElementTree Element
+
+    :param repodata_snippet: Snippet of XML to be turn into an ElementTree Element
+    :type  repodata_snippet: str
+
+    :return: Parsed ElementTree Element containing the parsed repodata snippet
+    :rtype:  xml.etree.ElementTree.Element
+    """
+    register_namespace('rpm', constants.RPM_NAMESPACE)
+    try:
+        # make a guess at the encoding
+        codec = 'UTF-8'
+        repodata_snippet.encode(codec)
+    except UnicodeEncodeError:
+        # best second guess we have, and it will never fail due to the nature
```

Comment:
This should be taken out, it can't be right. Hiding errors is an antipattern and in this case it is expressly wrong. There isn't any way it could be right. Please reconsider taking this out.